### ISSUE TITLE
Evaluating first stage autoencoders

### ISSUE THREAD

**Issue by leod** [CompVis/latent-diffusion#58]
First of all, thank you so much for making this high-quality repository as well as pretrained models publicly available! This is highly useful for exploring your research.

I am currently training first stage autoencoders on a custom dataset (SoundCloud images) and am struggling with evaluating these models (other than with the loss values logged in TensorBoard). I plan to compare the performance of initializing the autoencoder weights randomly vs. fine-tuning one of your pretrained autoencoders.

I would prefer to calculate rFID, PSNR, and PSIM the same way as you did for your [results table](https://github.com/CompVis/latent-diffusion#pretrained-autoencoding-models). Could you please provide a hint as to how you evaluate your autoencoders? Is there some other repository or toolkit that you rely on?

**Comment by wtliao** [comment]
@leod Hi, I want to do the same evaluation. Have you figured it out how to do that? Thanks!

**Comment by Eurus-Holmes** [comment]
@wtliao @leod @rromb FYI, I added some evaluation scripts for first stage autoencoders https://github.com/CompVis/latent-diffusion/pull/353/files

**Comment by wtliao** [comment]
> @wtliao @leod @rromb FYI, I added some evaluation scripts for first stage autoencoders https://github.com/CompVis/latent-diffusion/pull/353/files

It is really appreciate! I will try and discuss with you further!
